src/instana/__init__.py

```diff
@@ -1,38 +1,36 @@
 # coding=utf-8
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. 2016
 """
-▀████▀███▄   ▀███▀▄█▀▀▀█▄███▀▀██▀▀███     ██     ▀███▄   ▀███▀     ██
-  ██   ███▄    █ ▄██    ▀█▀   ██   ▀█    ▄██▄      ███▄    █      ▄██▄
-  ██   █ ███   █ ▀███▄        ██        ▄█▀██▄     █ ███   █     ▄█▀██▄
-  ██   █  ▀██▄ █   ▀█████▄    ██       ▄█  ▀██     █  ▀██▄ █    ▄█  ▀██
-  ██   █   ▀██▄█ ▄     ▀██    ██       ████████    █   ▀██▄█    ████████
-  ██   █     ███ ██     ██    ██      █▀      ██   █     ███   █▀      ██
-▄████▄███▄    ██ █▀█████▀   ▄████▄  ▄███▄   ▄████▄███▄    ██ ▄███▄   ▄████▄
+Instana
 
-https://www.instana.com/
+https://www.ibm.com/products/instana
 
-Documentation: https://www.instana.com/docs/
+Documentation: https://www.ibm.com/docs/en/instana-observability/current
 Source Code: https://github.com/instana/python-sensor
 """
 
-
+import importlib
 import os
 import sys
-import importlib
 
-from .version import VERSION
-from instana.collector.helpers.runtime import is_autowrapt_instrumented, is_webhook_instrumented
-
-__author__ = 'Instana Inc.'
-__copyright__ = 'Copyright 2020 Instana Inc.'
-__credits__ = ['Pavlo Baron', 'Peter Giacomo Lombardo', 'Andrey Slotin']
-__license__ = 'MIT'
-__maintainer__ = 'Peter Giacomo Lombardo'
-__email__ = 'peter.lombardo@instana.com'
+from instana.collector.helpers.runtime import (
+    is_autowrapt_instrumented,
+    is_webhook_instrumented,
+)
+from instana.version import VERSION
+
+__author__ = "Instana Inc."
+__copyright__ = "Copyright 2020 Instana Inc."
+__credits__ = ["Pavlo Baron", "Peter Giacomo Lombardo", "Andrey Slotin"]
+__license__ = "MIT"
+__maintainer__ = "Peter Giacomo Lombardo"
+__email__ = "peter.lombardo@instana.com"
```

Comment:
I think we should think about changing all the entries except `__license__` 😅

Reply:
We should also do that, but first, we need to investigate internally how to proceed in this case. 
For now, I'd like to keep it as is.